install.sh

```diff
@@ -94,14 +94,28 @@ nvm_download() {
 install_nvm_from_git() {
   local INSTALL_DIR
   INSTALL_DIR="$(nvm_install_dir)"
+  local NVM_VERSION
+  NVM_VERSION="${NVM_INSTALL_VERSION:-$(nvm_latest_version)}"
+  local IS_TAG
+  IS_TAG=0
+  [ "$(command git ls-remote "$(nvm_source "git")" "refs/tags/$NVM_VERSION" | wc -l)" != "0" ] && IS_TAG=1
```

Comment:
let's only run this when `NVM_INSTALL_VERSION` is provided:
```suggestion
  local IS_TAG
  IS_TAG=0
  if [ -z "${NVM_INSTALL_VERSION-}" ] || [ "$(command git ls-remote "$(nvm_source "git")" "refs/tags/$NVM_VERSION" | wc -l)" != "0" ]; then
    IS_TAG=1
  fi
```
(also, i prefer if/else over abusing flow control operators)

Reply:
done

Reply:
I change it a bit though, as by default, it should be a tag